File: sdkwork-backend-sdk-java/src/main/java/com/sdkwork/backend/model/QrCodeVO.java
package com.sdkwork.backend.model;

public class QrCodeVO {
    private String createdAt;
    private String updatedAt;
    private String qrKey;
    private String qrUrl;
    private String qrContent;
    private Integer expireTime;

    public String getCreatedAt() {
        return this.createdAt;
    }
    
    public void setCreatedAt(String createdAt) {
        this.createdAt = createdAt;
    }

    public String getUpdatedAt() {
        return this.updatedAt;
    }
    
    public void setUpdatedAt(String updatedAt) {
        this.updatedAt = updatedAt;
    }

    public String getQrKey() {
        return this.qrKey;
    }
    
    public void setQrKey(String qrKey) {
        this.qrKey = qrKey;
    }

    public String getQrUrl() {
        return this.qrUrl;
    }
    
    public void setQrUrl(String qrUrl) {
        this.qrUrl = qrUrl;
    }

    public String getQrContent() {
        return this.qrContent;
    }
    
    public void setQrContent(String qrContent) {
        this.qrContent = qrContent;
    }

    public Integer getExpireTime() {
        return this.expireTime;
    }
    
    public void setExpireTime(Integer expireTime) {
        this.expireTime = expireTime;
    }
}

File: sdkwork-backend-sdk-typescript/src/types/qr-code-vo.ts
/** 二维码生成响应 */
export interface QrCodeVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 二维码唯一标识 */
  qrKey?: string;
  /** 二维码图片URL */
  qrUrl?: string;
  /** 二维码内容(用于前端生成) */
  qrContent?: string;
  /** 过期时间(秒) */
  expireTime?: number;
}
